.github/workflows/post.yml

```diff
@@ -133,3 +132,13 @@ jobs:
           docker manifest push quay.io/open-cluster-management/work:latest
           docker manifest push quay.io/open-cluster-management/placement:latest
           docker manifest push quay.io/open-cluster-management/addon-manager:latest
+  trigger-clusteradm-e2e:
+    needs: [ images, image-manifest ]
+    name: trigger clusteradm e2e
+    runs-on: ubuntu-latest
+    steps:
+      - uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.OCM_BOT_PAT }}
+          repository: open-cluster-management-io/clusteradm
+          event-type: ocm_changes
```

Comment:
can we also send the commit sha, so clusteradm will know which commit triggers?

Reply:
Sure, the sha and ref added.